components/places/ffi/src/lib.rs

```diff
@@ -292,7 +292,7 @@ pub extern "C" fn places_delete_place(handle: u64, url: FfiStr<'_>, error: &mut
             }
         };
         if let Some(guid) = guid {
-            storage::history::delete_place_by_guid(conn, &guid)?;
+            storage::history::delete_all_visits_for_guid(conn, &guid)?;
```

Comment:
We probably should bite the bullet here and rename `places_delete_place` too - at least on Android, the "rename hell" only goes as far as one reference in a-c called `deleteVisitsFor()` so that makes sense. It looks like we don't have any iOS callers?